chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_VIEWS_TABS_BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_
+
+#define BRAVE_TAB_HOVER_CARD_BUBBLE_VIEW_H_ \
+                                            \
+ protected:                                 \
+  class BraveAnonymous;                     \
+                                            \
+ private:
+
+#define TabHoverCardBubbleView TabHoverCardBubbleView_ChromiumImpl
+#define UpdateCardContent virtual UpdateCardContent
```

Comment:
this could be used to get rid of a lot of other patches for `virtual`. I didn't realize it was valid to have `void virtual Method(...)` cc @bbondy

Reply:
@mkarolin please take advantage of this when doing chromium updates for existing patches using `virtual`

Reply:
Yes, you can have virtual after the return type as long as the return type is not a pointer or a reference. If they are then you need to have `A virtual *function`, which doesn't help us.

Reply:
that's the case even though the standard for chromium code is `T* function()`? Shouldn't the preprocessor replacement work correctly for that?